ExtendedCanvas/src/androidTest/java/com/microsoft/device/display/samples/extendedcanvas/ExtendedCanvasTest.kt

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.display.samples.extendedcanvas
+
+import androidx.compose.ui.graphics.asAndroidBitmap
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.captureToImage
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithContentDescription
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performGesture
+import androidx.compose.ui.test.swipeLeft
+import androidx.window.testing.layout.WindowLayoutInfoPublisherRule
+import com.microsoft.device.display.samples.extendedcanvas.ui.ExtendedCanvasAppsTheme
+import com.microsoft.device.dualscreen.testutils.compare
+import com.microsoft.device.dualscreen.testutils.getString
+import com.microsoft.device.dualscreen.testutils.zoomIn
+import com.microsoft.device.dualscreen.testutils.zoomOut
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.rules.TestRule
+
+class ExtendedCanvasTest {
+    private val composeTestRule = createAndroidComposeRule<MainActivity>()
+    private val publisherRule = WindowLayoutInfoPublisherRule()
+
+    @get: Rule
+    val testRule: TestRule
+
+    init {
+        testRule = RuleChain.outerRule(publisherRule).around(composeTestRule)
+        RuleChain.outerRule(composeTestRule)
+    }
```

Comment:
Since none of the tests use "simulateFold", you don't need to chain two rules together - this block of code can be replaced by the following:
```
@get: Rule
val composeTestRule = createAndroidComposeRule<MainActivity>()
```